CHANGES.txt

```diff
@@ -5,6 +5,57 @@ We only cover here the main highlights, for a detailed list of all the changes
 see `the github commits changelog
 <https://github.com/hyperspy/hyperspy/commits/master>`_.
 
+.. _changes_0.8.1:
+
+v0.8.1
+======
+
+This is a maintenance release. Follow the following links for details on all
+the `bugs fixed
+<https://github.com/hyperspy/hyperspy/issues?page=1&q=is%3Aclosed+milestone%3A0.8.1+label%3Abug&utf8=%E2%9C%93>`_,
+`feature
+<https://github.com/hyperspy/hyperspy/issues?utf8=%E2%9C%93&q=is%3Aclosed+milestone%3A0.8.1++label%3Aenhancement+>`_
+and `documentation
+<https://github.com/hyperspy/hyperspy/issues?q=is%3Aclosed+milestone%3A0.8.1+label%3Adocumentation>`_ enhancements.
+
+It also includes some new features and introduces important API changes that
+will be fully enforced in HyperSpy 0.9.
```

Comment:
``create_model`` is still there.  ``unfold_if_multidim`` is not. +1 to put it back with a VisibleDeprecationWarning. I'll do it.